### PR TITLE
[IMP] state: add model `Property` T#69469

### PR DESCRIPTION
**Pull request by Suavesito-Olimpiada**
@deivislaya 

This PR is ready for review, it implements section 4 of _Getting Started_.

Regards!

### CODE REVIEW

File: estate/models/estate_property.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate properties"
+
+    name = fields.Char(required=True)
```
Comment (deivislaya) [inline estate/models/estate_property.py]: Please, add help attribute in the field. Review all ocurrences

**Comment by deivislaya** [comment]
Please. Rebase this MR

File: estate/models/estate_property.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        [("north", "North"), ("south", "South"), ("east", "East"), ("west", "West")]
```
Comment (deivislaya) [inline estate/models/estate_property.py]:
```suggestion
        selection=[("north", "North"), ("south", "South"), ("east", "East"), ("west", "West")]
```

**Review by deivislaya — CHANGES_REQUESTED** [review]

File: estate/models/estate_property.py (hunk 1 above)
Comment (Suavesito-Olimpiada) [reply estate/models/estate_property.py]: Do I add help for all fields or just name? In #5 I add label string for those who need it, based on the model from the documentation.

File: estate/models/estate_property.py (hunk 2 above)
Comment (Suavesito-Olimpiada) [reply estate/models/estate_property.py]: Correction made.

File: estate/models/estate_property.py (hunk 1 above)
Comment (deivislaya) [reply estate/models/estate_property.py]: In all fields

Comment (deivislaya) [reply estate/models/estate_property.py]: change pending

Comment (Suavesito-Olimpiada) [reply estate/models/estate_property.py]: Solved.

**Review by deivislaya — APPROVED** [review]

**Pull request merged**